pom.xml

```diff
@@ -110,6 +98,54 @@
                      </includes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.10.0</version>
+                <configuration>
+                    <nodeVersion>v12.16.3</nodeVersion>
```

Comment:
Haven't used this plugin in a while, so this might be a dumb question. Is it needed to hardcode the node version like this, or can it be a bit more lenient (like resolving from the env, or pin to a major version?)

Reply:
It seems that they use this as part of the URL (https://nodejs.org/dist/) for downloading Node.js so full version is required. We could use pom variable and include it in `<properties>`... or any other suggestion?

Reply:
Making a property is probably the cleanest solution in terms of maintenance, although the LTS version won't change that often. So we can leave it as is.